plugins/system/apticron

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+# -*- sh -*-
+: <<=cut
+
+=head1 NAME
+
+apticron - Show the result of I<"apt-get -u dist-upgrade"> with four different graphs
+
+=head1 APPLICABLE SYSTEMS
+
+All systems with "apt-get" and "grep"
+
+=head1 CONFIGURATION
+
+The following is the default configuration
+
+  [apticron]
+  user root
+
+You could define two alert levels and the graph language
+
+  [apticron]
+  env.warning          [value]        (default:  80)
+  env.critical         [value]        (default: 100)
+  env.language         [en|de|es]     (default:  en)
+
+or if you like define more details
+
+  env.warning_update   [value]        (default:  80 or env.warning, if set)
+  env.warning_newone   [value]        (default:  25% of env.warning_update)
+  env.warning_remove   [value]        (default:   2% of env.warning_update)
+  env.warning_holdit   [value]        (default:   2% of env.warning_update)
+
+  env.critical_update  [value]        (default: 100 or env.critical, if set)
+  env.critical_newone  [value]        (default:  25% of env.critical_update)
+  env.critical_remove  [value]        (default:   2% of env.critical_update)
+  env.critical_holdit  [value]        (default:   2% of env.critical_update)
+
+=head1 DESCRIPTION
+
+This plugin is a graphic version of die I<apicron package>, but it sends no mails.
+
+It showns graphs for
+
+  - upgraded
+  - newly installed
```

Comment:
If I understand your interpretation of `apt-get`'s output correctly, then this number just describes, how many "new" packages appeared in the local package list after the last run of `apt-get update`. Are you sure, that you want to measure this state?
If yes: how could you describe it more precisely?